portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,51 +29,45 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.Arrays;
-import java.util.ArrayList;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns comments*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-    ArrayList<String> words = new ArrayList<String>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    Query query = new Query("Task").addSort("timestamp", SortDirection.DESCENDING);
-    
+    //query for comment storage through datastore.
+    Query query = new Query("Task");
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
     
-    //iterating over entities within Datastore
+    //populate comments ArrayList with entities within Datastore.
```

Comment:
Maybe : "Populate comments with titles from datastore."

Rationale:

- We don't need to know comments is an ArrayList (it's already there in the code).
- entities is somewhat vague, mentioning "title" itself might be good enough.